src/placeos-rest-api/controllers/systems.cr

```diff
@@ -28,6 +28,11 @@ module PlaceOS::Api
 
     id_param :sys_id
 
+    # Allow unscoped access to details of a single `ControlSystem`
+    before_action :can_read, only: [:index]
+    before_action :can_guest_read, only: [:show, :sys_zones]
+    before_action :can_write, only: [:create, :update, :destroy, :remove, :update_alt]
```

Comment:
I think this needs further review to ensure all routes are scoped

Reply:
Should we add an Execute scope? For `:control, :execute...` etc